docs/source/resources/remotekubernetesclusters.md

```diff
@@ -0,0 +1,57 @@
+# RemoteKubernetesCluster
+
+## Introduction
+
+The cluster-scoped `RemoteKubernetesCluster` resource provides an abstraction layer for managing resources in a remote Kubernetes cluster.
+
+It allows users to define and interact with a separate Kubernetes environment, distinct from the cluster where the resource is created.  
+This is particularly useful in scenarios that require centralized management of ScyllaDB clusters distributed across multiple geographic regions.
+
+## Example
+
+```{literalinclude} ../../../examples/multi-dc/cluster-wide-resources/00_dev-us-east-1.remotekubernetescluster.yaml
+:language: yaml
+:linenos:
+```
+
+## Remote authorization methods
+
+### Secret
+
+This authorization method relies on a Secret containing credentials to a remote Kubernetes cluster in the _kubeconfig_ format.  
+The user associated with these credentials must have the `scylladb:controller:operator-remote` ClusterRole bound.
+The name and namespace of this Secret are referenced in the spec of the corresponding RemoteKubernetesCluster resource.
+
+Example:
+
+::::{caution}
```

Comment:
To implement the decision we've reached regarding https://github.com/scylladb/scylla-operator/issues/2577 -

Please add a notice similar to the below:

```
::::{caution}
Currently, `RemoteKubernetesCluster` requires unconditional access to all `Secrets` and `ConfigMaps` across all namespaces in the remote cluster. Please note the [security implications](https://github.com/scylladb/scylla-operator/issues/2577) of this limitation.
::::
```

Reply:
added